schemas/json/layout/layout.schema.v1.json

```diff
@@ -41,7 +41,7 @@
             "type": "string",
             "title": "Type",
             "description": "The component type.",
-            "enum": ["AddressComponent", "Button", "Checkboxes", "Datepicker", "Dropdown", "FileUpload", "Group", "Header", "Input", "NavigationButtons", "Paragraph", "RadioButtons", "Summary", "TextArea"]
+            "enum": ["AddressComponent", "Button", "Checkboxes", "Datepicker", "Dropdown", "FileUpload", "Group", "Header", "Input", "NavigationButtons", "Paragraph", "RadioButtons", "Summary", "TextArea", "AttachmentList"]
```

Comment:
Should be second in the list (alphabetical order).

Reply:
Fixed :)